common/components/WalletDecrypt/Mnemonic.tsx

```diff
@@ -19,14 +20,15 @@ interface State {
 export default class MnemonicDecrypt extends Component<Props, State> {
   public state: State = {
     phrase: '',
+    formattedPhrase: '',
     pass: '',
     seed: '',
     dPath: DEFAULT_PATH
   };
 
   public render() {
-    const { phrase, seed, dPath, pass } = this.state;
-    const isValidMnemonic = validateMnemonic(phrase);
+    const { phrase, formattedPhrase, seed, dPath, pass } = this.state;
```

Comment:
can remove phrase from de-structure (unless I'm missing something further down)

Reply:
it's used at least here: https://github.com/MyEtherWallet/MyEtherWallet/blob/develop/common/components/WalletDecrypt/Mnemonic.tsx#L41